docs/asyncapi.yaml

```diff
@@ -0,0 +1,126 @@
+asyncapi: 3.0.0
+info:
+  title: API contract book for Single Front Door user communications
+  version: 1.0.0
+  description: Publish communications to users via the Single Front Door
+
+channels:
+  fcp-fd-comms:
+    address: fcp-fd-comms-request
+    description: |
+      Accepts requests for communication notifications via Azure service bus.
+    messages:
+      notifyMessage:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/fcp-fd-comms'
+    
+
+components:
+  messages:
+    notifyMessage:
+      title: gov.notify communications message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/govNotifyMessage'
+
+  schemas:
+    govNotifyMessage:
+      type: object
+      properties:
```

Comment:
Could consider breaking these down into separate schemas referenced here.  Then the structure is easier to read and if folks want the detail about fields, they can jump to it.

Reply:
You mean take out the business logic aka  "data" into it's own schema?